tests/foreman/ui/test_repository.py

```diff
@@ -942,10 +942,10 @@ def test_positive_recommended_repos(session, module_entitlement_manifest_org):
         cap_tool_repos = [
             repo['name']
             for repo in rrepos_on
-            if 'Tools' in repo['name'] or 'Capsule' in repo['name']
+            if 'Utils' in repo['name'] or 'Capsule' in repo['name']
         ]
         cap_tools_repos = [repo for repo in cap_tool_repos if repo.split()[4] != sat_version]
-        assert not cap_tools_repos, 'Tools/Capsule repos do not match with Satellite version'
```

Comment:
during debugging I got `cap_tools_repos` emtpy list []
what is intention behind assertion? can we add some comment statement before this assertion to make it more clear?

Reply:
@ColeHiggins2 Could you please add some meaningful comment statement before assertion statement of line 948 ?

Reply:
That is a very good point @vijaysawant. I favor changing the assertion entirely and eventually changing the code above it.
Can't we simply check that the specific repositories are not listed using a repository name template? I assume that the repository names have the same format across Satellite releases. If that format changes, changing the format in the test is an easy fix.

Reply:
@vijaysawant Your empty list was due to the fact that the expected recommended repos are not accessible yet. I assume you tested on 6.15